ppafm/cli/gui/ppafm_gui.py

```diff
@@ -577,9 +609,14 @@ def loadInput(self, main_input, rho_sample=None, rho_tip=None, rho_tip_delta=Non
         if self.rho_sample is None:
             self.bxV0.setDisabled(True)
             self.bxAlpha.setDisabled(True)
+            self.bxCutoffEnable.setDisabled(True)
+            self.bxCutoffEnable.setChecked(False)
+            self.bxCutoffValue.setDisabled(True)
         else:
             self.bxV0.setDisabled(False)
             self.bxAlpha.setDisabled(False)
+            self.bxCutoffEnable.setDisabled(False)
+            self.previous_cutoff = None
```

Comment:
Yes, it's so that we know on each update if the cutoff has changed from the previous value. If it has changed then we apply the cutoff and redo the FFT on the tip density array. It's basically just to prevent redoing the same computation multiple times.